workspace.json

```diff
@@ -8008,7 +8008,7 @@
       "prefix": "personal-representative",
       "architect": {
         "build": {
-          "builder": "@nrwl/node:build",
+          "builder": "@irdnis/esbuildnx:build",
```

Comment:
For both services you need to add `esbuild.json`  file to list external dependencies.
You can read more about it [here](https://esbuild.github.io/api/#external).

You can find examples of `esbuild.json` from other services in the repo.
Like [this one](https://github.com/island-is/island.is/blob/main/apps/services/auth-api/esbuild.json)

Reply:
Just added :)